plugins/filter_aws/aws.c

```diff
@@ -102,17 +100,33 @@ static void expose_aws_meta(struct flb_filter_aws *ctx)
                     "aws." FLB_FILTER_AWS_HOSTNAME_KEY,
                     ctx->hostname);
     }
+
+    for (i = 0; i < ctx->tags_count; i++) {
+        /* len("aws.ec2.tags.") = 13 */
+        env_tag_name = flb_sds_create_size(13 + ctx->tag_keys_len[i]);
+        if (!env_tag_name) {
+            flb_errno();
+            continue;
+        }
+        env_tag_name = flb_sds_printf(&env_tag_name, "aws.ec2.tags.%s", ctx->tag_keys[i]);
+        if (!env_tag_name) {
+            flb_errno();
```

Comment:
Shouldn't we also free the memory allocated for `env_tag_name` (at line 106) before `continue`?

Reply:
Fixed, https://github.com/fluent/fluent-bit/pull/6065/commits/464daecd448f6124328567810870514f16d7673e